alf/networks/encoding_networks.py

```diff
@@ -659,6 +682,13 @@ def __init__(self,
                 used.
             use_fc_bn (bool): whether use Batch Normalization for fc layers.
             use_fc_ln (bool): whether use Layer Normalization for fc layers.
+            use_batch_ensemble (bool): whether use BatchEnsemble FC and Conv
```

Comment:
should comment the output of the network, i.e., it is tuple with ensemble_ids

Reply:
Updated.